src/CTA.Rules.Models/Tokens/NodeToken.cs

```diff
@@ -46,6 +47,7 @@ public NodeToken()
         public List<UsingAction> UsingActions { get; set; }
         public List<IdentifierNameAction> IdentifierNameActions { get; set; }
         public List<InvocationExpressionAction> InvocationExpressionActions { get; set; }
+        public List<ExpressionAction> ExpressionActions { get; set; }
```

Comment:
Could you rebase main branch, we recently [changed NodeToken.Clone](https://github.com/aws/cta/pull/155/files#diff-869111198b1aa0c5661f152367a3fbf7da1332f678f4389d852a10fa94427c03R55) from shallow to deep clone, the `ExpressionActions` list should be deep cloned too.